### ISSUE TITLE
Carve bad again?

### ISSUE THREAD

**Issue by Karaulov** [LogicAndTrick/sledge#278]
Before:
![di-3kudsr 1](https://user-images.githubusercontent.com/6445336/47515598-f8bd2800-d88b-11e8-889f-a0c7462769c1.png)

After:
![liusobw 1](https://user-images.githubusercontent.com/6445336/47515614-04a8ea00-d88c-11e8-827c-b97ba8a56436.png)

Map example:
[supertbadcarver.zip](https://github.com/LogicAndTrick/sledge/files/2515908/supertbadcarver.zip)

**Comment by LogicAndTrick** [comment]
Yes, carve is bad, you shouldn't use it. Especially on two circular objects.

**Comment by Karaulov** [comment]
@LogicAndTrick You do not want to make a good editor with perfect working tools? :(

Then you can make a warning that Carve will not work with these objects?  :)

**Comment by LogicAndTrick** [comment]
I probably shouldn't have closed it as it is a bug, but don't expect it to be fixed any time soon. Please stop using carve on objects like this.

**Comment by Karaulov** [comment]
:(

But I do not know how to make map without Carve :) This tool would be well suited for creating some bad maps :)
